docs/install_occlum_packages.md

```diff
@@ -0,0 +1,99 @@
+# Install Occlum Packages with Package Mangement Tool
+
+Occlum packages can be easily installed and deployed using mainstream package management tool including RPM and APT.
+
+
+### Prerequisite
+
+**Install enable_RDFSBASE Kernel Module**
+
+Please follow [this README](https://github.com/occlum/enable_rdfsbase/blob/master/README.md) to install `enable_rdfsbase` kernel module.
+
+
+### Install Occlum with RPM
+
+1. Install Prerequisite
+```
+yum install -y wget yum-utils make jq gdb
+```
+
+2. Install Intel® SGX Driver and Intel® SGX PSW
+
+Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.11/docs/Intel_SGX_Installation_Guide_Linux_2.11_Open_Source.pdf) to install SGX driver and SGX PSW. SGX SDK is not required. Using RPM installer is recommanded. 
+
+Also, UAE service libraries are needed but may not installed together with SGX PSW if SGX PSW installer is used. Go to SGX RPM local repo and run:
+```
+rpm -i libsgx-uae-service*.rpm
+```
+
+After installing SGX driver and PSW, please make sure `aesm` service is running by checking:
+```
+service aesmd status
+```
+
+3. Install Occlum
+```
+cat << EOF > /etc/yum.repos.d/occlum.repo
+[occlum]
+name=occlum
+enabled=1
+baseurl=https://occlum.io/occlum-package-repos/rpm-repo/
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://occlum.io/occlum-package-repos/rpm-repo/RPM-GPG-KEY-rpm-sign
+gpgcakey=https://occlum.io/occlum-package-repos/rpm-repo/RPM-GPG-KEY-rpm-sign-ca
+EOF
+yum --showduplicate list -y occlum
+yum install -y occlum
+echo "source /etc/profile" >> $HOME/.bashrc
+```
+
+
+### Install Occlum with APT
+
+1. Install Prerequisite
+```
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends ca-certificates gnupg2 jq make gdb wget libfuse-dev libtool
```

Comment:
If user only want to run the occlum instance, user does not need all those dependents. Should we separate this list to two?

Reply:
I think these are the minimal dependencies. wget certificate and gnupg are used for package installation. Others are needed by occlum commands.